src/contentful/utils.ts

```diff
@@ -42,16 +61,22 @@ export const getContentfulRequest = (type, contentQuery) => {
   };
 };
 
-const hasValidationErrors = (payload) => payload?.errors;
+const hasValidationErrors = (payload?: Record<string, unknown>) => payload?.errors;
 
-export const createContentfulRequest = async(type, contentQuery) => {
+export const createContentfulRequest = async (
+  type: string,
+  contentQuery: Record<string, unknown>,
+) => {
```

Comment:
missing return type

we already have `ContentfulResponseObject` available - is that sufficient?